spec/dummy/app/controllers/dummy_models_controller.rb

```diff
@@ -44,4 +44,8 @@ def index_custom_formatter
   def index_group_by
     render_paginated DummyModel.ocurrences_of_name
   end
+
+  def index_custom_adapter
+    render_paginated DummyModel, adapter: CustomAdapter
+  end
```

Comment:
Could you add this example to readme?

Reply:
In the next commit there's a whole complete explanation on how to operate with CustomAdapters in the README